RELEASE/scripts/sl_ascend.ash

```diff
@@ -14365,9 +14365,9 @@ boolean doTasks()
 	if(Lsc_flyerSeals())				return true;
 	if(L11_blackMarket())				return true;
 	if(L11_forgedDocuments())			return true;
+	if(L11_mauriceSpookyraven())		return true;
```

Comment:
Should be one line lower, under L11_mcmuffinDiary, I think.

Reply:
Oh, right, can't access until you have the diary. Good catch, thank you!

Reply:
Also, there are two copies of these lines:

```
if(L11_blackMarket())				return true;
if(L11_forgedDocuments())			return true;
if(L11_mcmuffinDiary())				return true;
```

Should the upper chunk be deleted, or the lower...?

Reply:
Hah, actually, that explains why the code worked as-is. The upper `L11_mcmuffinDiary()` call got the diary before reaching this line.

Reply:
Humm, it's strange that there's an additional copy of those lines... Seems like the lower copy should be deleted, but it also seems like such a weird thing that there might be a good reason for it?

Reply:
Yeah, I'm content to just back away slowly at this point, haha. It's harmless enough as-is.